# LitterList.msg
bumperbot_detection/LitterPoint[] litter_points
